chapter12/sxg/ex_12_2.pl

```diff
@@ -0,0 +1,23 @@
+#!perl
+use warnings;
+use strict;
+
+die "No file names found!\n" unless @ARGV;
+
+my $old_name = shift @ARGV;
+my $old_age = -M $old_name;
+$old_age = 0 if (!defined $old_age) ;

```

Comment:
I saw "Inode-modification age" and got modification. Didn't grasp that it would give you creation date